cachi2/core/package_managers/yarn/main.py

```diff
@@ -109,6 +111,23 @@ def _configure_yarn_version(project: Project) -> None:
         project.package_json.write()
 
 
+def _get_allowed_plugins(yarn_rc: YarnRc) -> list[Plugin]:
+    """Return a list of plugins that can be kept in .yarnrc.yml.
+
+    The plugins returned here are added by default by 'yarn init', or are included by default in
+    yarn 4+. They are not safe by default, but Cachi2 handles them in a safe way.
```

Comment:
Also, we should make it more clear in the docstring that plugins returned by this helper are solely for means of being able to run `yarn info` successfully.